packages/contracts-ops/contracts/scripts/UpgradeCallBatches.s.sol

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+/*//////////////////////////////////////////////////////////////
+                                 IMPORTS
+    //////////////////////////////////////////////////////////////*/
+
+// Utilities
+import {Test} from "forge-std/Test.sol";
+import {console2} from "forge-std/console2.sol";
+
+// Ops libs
+import {Config} from "../Config.sol";
+import {CallBatch} from "../CallBatch.sol";
+
+contract UpgradeCallBatches is Test, Config, CallBatch {
+    /*//////////////////////////////////////////////////////////////
+                                 BEACONS
+    //////////////////////////////////////////////////////////////*/
+
+    address homeBeacon;
+    address replicaBeacon;
+    address governanceRouterBeacon;
+    address bridgeRouterBeacon;
+    address tokenRegistryBeacon;
+    address bridgeTokenBeacon;
+
+    /*//////////////////////////////////////////////////////////////
+                             IMPLEMENTATIONS
+    //////////////////////////////////////////////////////////////*/
+
+    address homeImpl;
+    address replicaImpl;
+    address governanceRouterImpl;
+    address bridgeTokenImpl;
+    address tokenRegistryImpl;
+    address bridgeRouterImpl;
+
+    /*//////////////////////////////////////////////////////////////
+                            GOVERNANCE CALLS
+    //////////////////////////////////////////////////////////////*/
+
+    bytes upgradeHome;
+    bytes upgradeReplica;
+    bytes upgradeGovRouter;
+    bytes upgradeBridgeRouter;
+    bytes upgradeTokenRegistry;
+    bytes upgradeBridgeToken;
+
+    address beaconController;
+    address governanceRouterProxy;
+
+    bytes executeCallBatchCall;
+
+    string configFile;
+    string[] domainNames;
+    string[] networksArray;
+
+    function run(
```

Comment:
more informative fn name would be ideal

Reply:
This is a best practice for forge scripts that we try to make a standard, that is `run` is the entrypoint. I would prefer to keep it, but I can change it if y'all insist

Reply:
okay, changed